source/ml_methods/SklearnMethod.py

```diff
@@ -49,19 +49,19 @@ class SklearnMethod(MLMethod):
                     # Additional parameter that determines whether to print convergence warnings
                     show_warnings: True
                 # if any of the parameters under SimpleLogisticRegression is a list and model_selection_cv is True,
-                # SimpleLogisticRegression will do grid search over the given parameters and return optimal model
-                model_selection_cv: False
-                model_selection_n_folds: -1
+                # a grid search will be done over the given parameters, using the number of folds specified in model_selection_n_folds,
+                # and the optimal model will be selected
+                model_selection_cv: True
+                model_selection_n_folds: 5
             svm_with_cv:
                 SVM: # name of another class inheriting SklearnMethod
                     # sklearn parameters (same names as in original sklearn class)
                     alpha: [10, 100] # search will be performed over these parameters
                     # Additional parameter that determines whether to print convergence warnings
                     show_warnings: True
-                # if any of the parameters under SVM is a list and model_selection_cv is True,
-                # SVM will do grid search over the given parameters and return optimal model
-                model_selection_cv: True
-                model_selection_n_folds: 5
+                # no grid search will be done
```

Comment:
this will fail if run: if any of the parameters under class name is a list, model_selection_cv has to be True and model_selection_n_folds has to be equal to or larger than 2 :)

Reply:
updated now!